src/python/pants/init/plugin_resolver.py

```diff
@@ -80,7 +80,6 @@ async def resolve_plugins(
             internal_only=True,
             python=python,
             requirements=requirements,
-            interpreter_constraints=request.interpreter_constraints,
```

Comment:
This was `None` and thats a no-no. However there's no type hint for this param so `mypy` isn't complaining.